installer/templates/new/widgets/number/number.js

```diff
@@ -36,15 +36,24 @@ Widget.mount(class Number extends Widget {
       return '';
     }
   }
+  changeRate() {
+    if (this.props.changerate == "off") {
```

Comment:
I'll update with an early return. I can't do a test of `this.props.changerate === false` because the property value is not a boolean, it's a string. My reason for using on/off instead of true/false as the property value was so that future devs don't get confused thinking that the property value is a boolean value.